### PR TITLE
first draft Busola configuration

### PR DESCRIPTION
**Pull request by NHingerl**
**Description**

Changes proposed in this pull request:

- add draft document based on slack conversation
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: CONFIGURE_BUSOLA.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+| `navigation.externalNodes` | A nested list of links to external websites. `category`, `icon`: a category name and optional icon / `children`: a list of pairs (label and link) | category: Learn more / children: Kyma Documentation, Our Slack, Github                                                                                                                                               |
+| `version`                  | Configuration version. Don’t edit this. Can be empty.                                                                                             | the most recent release                                                                                                                                                                                              |
+
+## Change the Configuration
+
+If you have the required authorizations and access to the kubeconfig, you can change the settings for the Busola cluster configuration and the target cluster configuration.
```
[the remark this answers is not in the document]
Comment (NHingerl) [reply CONFIGURE_BUSOLA.md:18]: why not?

File: CONFIGURE_BUSOLA.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+With the `feature` toggles, you can switch each Busola feature on or off.
+There are two kinds of features:
+
+- KUBECONFIG_ID: If enabled, and if a kubeconfig ID is defined, it pulls the kubeconfig from the service specified at `config.kubeconfigUrl`.
```
[the remark this answers is not in the document]
Comment (NHingerl) [reply CONFIGURE_BUSOLA.md]: ✔️

File: CONFIGURE_BUSOLA.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+- The remaining features, which contain the `apiGroup` selector. They enable and disable navigation nodes on the UI and some functionalities. For example, if `API_GATEWAY` is enabled, the `API Rules` node is displayed, and the Function details allow exposing a Function with API rules.
```
[the remark this answers is not in the document]
Comment (NHingerl) [reply CONFIGURE_BUSOLA.md]: ✔️  Perfect, thank you!

**Review by valentinvieriu — APPROVED** [review]

File: CONFIGURE_BUSOLA.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# Configure Busola
+
```
[the remark this answers is not in the document]
Comment (NHingerl) [reply CONFIGURE_BUSOLA.md:2]: it makes total sense! thanks for spotting this!

File: CONFIGURE_BUSOLA.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# Configure Busola
```
[the remark this answers is not in the document]
Comment (NHingerl) [reply CONFIGURE_BUSOLA.md:1]: I'm absolutely not sure :D I'll align with Valentin.

[the remark this answers is not in the document]
Comment (NHingerl) [reply CONFIGURE_BUSOLA.md]: renamed it, will move the content when I know Valentin's preference

File: CONFIGURE_BUSOLA.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+| `features`                 | Switches a set of Busola features on and off. Use selectors to configure conditions for the features. To switch them off, set `isEnabled=false`.    | isEnabled=true                                                                                                                                                                                                       |
+| `navigation.disabledNodes` | Array of IDs of navigation nodes that are hidden from navigation. Format: `<category>.<nodeName>` or `<namespace>.<category>.<nodeName>`)             | empty                                                                                                                                                                                                                |
+| `navigation.externalNodes` | A nested list of links to external websites. `category`, `icon`: a category name and optional icon / `children`: a list of pairs (label and link) | category: Learn more / children: Kyma Documentation, Our Slack, Github                                                                                                                                               |
```
[the remark this answers is not in the document]
Comment (NHingerl) [reply CONFIGURE_BUSOLA.md]: done

**Review by valentinvieriu — DISMISSED** [review]

**Comment by NHingerl** [comment]
@valentinvieriu Thanks for your approval. Please confirm that you want to keep the doc where it is or instead follow [Aleks' suggestion](https://github.com/kyma-project/busola/pull/364#discussion_r712352234) to create a docs folder or incorporate the content in the README.

File: CONFIGURE_BUSOLA.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (NHingerl) [reply CONFIGURE_BUSOLA.md]: It's in the README now :)

**Review by alexandra-simeonova — APPROVED** [review]

**Pull request merged**